### DISCUSSION TITLE
How to use stopwatch in web applications with egui?

### DISCUSSION THREAD

**Discussion by gravit22** [emilk/egui#299] [Q&A] [answered]
When I am trying to use wasm-stopwatch it give me this:
`error[E0277]: the trait bound `Stopwatch: Deserialize<'_>` is not satisfied
  --> src/app.rs:15:5
   |
15 |     stopwatch: Stopwatch,
   |     ^^^^^^^^^ the trait `Deserialize<'_>` is not implemented for `Stopwatch`
   |
   = note: required by `next_element`
`
But how to derive Deserialize for external crates? 
P. S. I tried https://serde.rs/remote-derive.html, but it dosn't work or I do not how to set up it for template project.

**Comment by emilk** [comment, marked as answer]
Just add `#[serde(skip)]` to skip serializing a field (or `#[cfg_attr(feature = "persistence", serde(skip))]` if you carbon-copied `egui_template` and kept the `persistence` feature flag)

**Reply by gravit22** [reply]
Thank you. That works completely fine.
